Add Aside component (#761)
withastro/docs: src/pages/en/guides/styling.md

This is not recommended for most users. Instead, place your CSS files inside of `public/` to get a consistent URL reference.

:::caution

> Importing a smaller CSS file with `?url` may return the base64 encoded contents of the CSS file as a data URL, but only in your final build. You should either write your code to support encoded data URLs (`data:text/css;base64,...`) or set the [`vite.build.assetsInlineLimit`](https://vitejs.dev/config/#build-assetsinlinelimit) config option to `0`  to disable this feature.

Reviewer: ```suggestion
Write your code to support encoded data URLs (`data:text/css;base64,...`) or set the [`vite.build.assetsInlineLimit`](https://vitejs.dev/config/#build-assetsinlinelimit) config option to `0` in order to avoid returning the base64 encoded contents of the CSS file as a data URL in your final build.
```

Author: I find this less clear than the current text. Current version is “small css can cause base64 URL, so…” whereas this change would leave a vague threat of base64 causing a problem. I’d like to keep this closer to the current version if you don’t mind? Here’s a slightly slimmer version keeping the current structure:

```suggestion
Importing a smaller CSS file with `?url` may return the base64 encoded contents of the CSS file as a data URL in your final build. Either write your code to support encoded data URLs (`data:text/css;base64,...`) or set the [`vite.build.assetsInlineLimit`](https://vitejs.dev/config/#build-assetsinlinelimit) config option to `0`  to disable this feature.
```

Reviewer: Aww... no vague threat? OK!
